src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesSlave.java

```diff
@@ -538,6 +543,51 @@ public static Builder builder() {
         return new Builder();
     }
 
+    public void annotateTtl(TaskListener listener) {
+        var kubernetesCloud = getKubernetesCloud();
```

Comment:
Can this be null?

Reply:
Raises an `IllegalStateException` if so, which is caught in order to avoid breaking the whole loop.